streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImplTest.java

```diff
@@ -230,6 +240,112 @@ public void testTotalMetricDoesntDecrease() {
             assertEquals(i, Math.round(totalMetric.measurable().measure(config, time.milliseconds())));
             sensor.record(latency, time.milliseconds());
         }
+    }
+
+    @Test
```

Comment:
This is a meta comment: for SampledStat their recorded values can vary based on the windowing itself, so in unit test it may not always has the `exact` value, i.e. asserting `equalTo(value)` may make this flaky, whereas just testing it is non-zero is sufficient.

Reply:
If we are using `MockTime` then it should be fine.